src/aap_eda/services/ruleset/podman_ruleset_handler.py

```diff
@@ -0,0 +1,71 @@
+#  Copyright 2023 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import logging
+
+from django.conf import settings
+
+from aap_eda.core import models
+
+from .activation_db_logger import ActivationDbLogger
+from .activation_podman import ActivationPodman
+from .deactivation_podman import DeactivationPodman
+from .ruleset_handler import RulesetHandler
+
+logger = logging.getLogger(__name__)
+
+
+class PodmanRulesetHandler(RulesetHandler):
+    def activate(
+        self,
+        ws_url: str,
```

Comment:
@hsong-rh Can't we get the ws_url and ssl_verify from the settings we don't have to pass those around. At the point where we use it to generate the command line for activation we can use it directly from settings. The command line for ansible-rulebook is going to be the same for both podman and k8s.